pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -539,7 +539,7 @@ func GetOpenshiftBootstrapClusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule("get").Groups(kapiGroup).Resources("pods/log").RuleOrDie(),
 				rbacv1helpers.NewRule("create", "list").Groups(kapiGroup).Resources("events").RuleOrDie(),
 
-				rbacv1helpers.NewRule("update").Groups(imageGroup, legacyImageGroup).Resources("imagestreamtags").RuleOrDie(),
+				rbacv1helpers.NewRule("create", "update").Groups(imageGroup, legacyImageGroup).Resources("imagestreamtags").RuleOrDie(),
```

Comment:
@openshift/sig-auth what did we get with 1.12 rebase that could have caused the need for this change? @bparees or did something changed wrt to imagestream?

@soltysh (this is the actual fix for the rebase revert)

Reply:
nothing has changed related to imagestream permissions from our side.